microservices/submission-service/src/replay-validation/replay-validation.service.ts

```diff
@@ -127,11 +127,24 @@ export class ReplayValidationService {
             accountType: "DISCORD",
             accountId: p.discordId,
         })));
-        if (userIdsResponses.some(r => r.status === ResponseStatus.ERROR)) {
-            this.logger.error(`Unable to validate submission, couldn't map from MLE player to Sprocket user by discordId`, JSON.stringify(userIdsResponses));
+
+        if (userIdsResponses.some(r => r.status == ResponseStatus.ERROR)) {
+
+            // Build up a list of failed player Discord IDs to return in the error response.
+            var errors: ValidationError[] = [];
+            userIdsResponses.forEach((r,i) => {
+                if (r.status == ResponseStatus.ERROR) {
+                    errors.push({
+                        error: `Could not find a Sprocket account for ${players[i].discordId}`,
+                        playerIndex: i,
+                    });
+                }
+            });
+
+            this.logger.error(`Unable to validate submission, couldn't map from MLE player(s) to Sprocket user by discordId:`, JSON.stringify(userIdsResponses));
```

Comment:
I wouldn't stringify here. Mapping and joining into a string is cleaner/easier to read.